frontend2/src/components/elements/Button.tsx

```diff
@@ -28,12 +28,12 @@ interface ButtonProps extends React.ComponentPropsWithoutRef<"button"> {
 
 const Button: React.FC<ButtonProps> = ({
   variant = "",
-  disabled = false,
   loading = false,
   label,
   iconName,
   fullWidth = false,
   className = "",
+  disabled,
```

Comment:
oops it looks like your merge commit removed the `disabled=false` here which is necessary for eslint 
```suggestion
  disabled = false,
```